build/main.tf

```diff
@@ -15,20 +15,21 @@ provider "azurerm" {
 locals {
   workbook_data_json_for_community = var.deploy_community_edition_to_azure ? templatefile("${path.module}/templates/workbook.tpl.json", {
     // load workbook already deployed on Azure subscription or community GitHub repo
-    "summary_workbook_resource_id"           = azurerm_application_insights_workbook.summary[0].id
-    "azuresiterecovery_workbook_resource_id" = azurerm_application_insights_workbook.azuresiterecovery[0].id
-    "compute_workbook_resource_id"           = azurerm_application_insights_workbook.compute[0].id
-    "containers_workbook_resource_id"        = azurerm_application_insights_workbook.containers[0].id
-    "databases_workbook_resource_id"         = azurerm_application_insights_workbook.databases[0].id
-    "integration_workbook_resource_id"       = azurerm_application_insights_workbook.integration[0].id
-    "networking_workbook_resource_id"        = azurerm_application_insights_workbook.networking[0].id
-    "storage_workbook_resource_id"           = azurerm_application_insights_workbook.storage[0].id
-    "web_workbook_resource_id"               = azurerm_application_insights_workbook.web[0].id
-    "servicealert_workbook_resource_id"      = azurerm_application_insights_workbook.servicealert[0].id
+    "summary_workbook_resource_id"             = azurerm_application_insights_workbook.summary[0].id
+    "azuresiterecovery_workbook_resource_id"   = azurerm_application_insights_workbook.azuresiterecovery[0].id
+    "compute_workbook_resource_id"             = azurerm_application_insights_workbook.compute[0].id
+    "containers_workbook_resource_id"          = azurerm_application_insights_workbook.containers[0].id
+    "databases_workbook_resource_id"           = azurerm_application_insights_workbook.databases[0].id
+    "integration_workbook_resource_id"         = azurerm_application_insights_workbook.integration[0].id
+    "networking_workbook_resource_id"          = azurerm_application_insights_workbook.networking[0].id
+    "storage_workbook_resource_id"             = azurerm_application_insights_workbook.storage[0].id
+    "web_workbook_resource_id"                 = azurerm_application_insights_workbook.web[0].id
+    "servicealert_workbook_resource_id"        = azurerm_application_insights_workbook.servicealert[0].id
+    "resiliencyscenarios_workbook_resource_id" = azurerm_application_insights_workbook.resiliencyscenarios[0].id
```

Comment:
It is better to rename this to something similar to "resiliencyscenario_multiregion" or "multiregionscenario" because there might be new workbooks supporting other resiliency scenarios in future. Feel free to use your own name but should be able to include additional resiliency scenarios without renaming this workbook name.